packages/orbit-components/src/Table/TableBody/index.tsx

```diff
@@ -1,27 +1,11 @@
-"use client";
-
 import * as React from "react";
-import styled from "styled-components";
 
 import type * as Common from "../../common/types";
-import defaultTheme from "../../defaultTheme";
 
-export const StyledTableBody = styled(({ children, className, dataTest }) => (
-  <tbody className={className} data-test={dataTest}>
+const TableBody = ({ children, dataTest }: React.PropsWithChildren<Common.Globals>) => (
+  <tbody className="border-b-table w-full whitespace-nowrap border-b" data-test={dataTest}>
```

Comment:
for maintainability, we should strive to utilise sensible defaults and cascading over overly specifying styles. I'd avoid specifying styles we don't need to write explicitly

it would also open a Pandora's box — which styles do we _not_ specify? we would have to specify e.g. `box-border block` on every `<div>`

Reply:
I agree overall. I was just following @mainframev's practice, as he's adding some styles in case preflight is disabled. And since this case, it was explicit in styled-components I thought we could include it. The search team is facing some issues because of this very same reason, but it should be their responsibility.

Reply:
yep that's the point. we can't consider every wild thing consumers are doing 😄 we should to write the minimum we need, and let the rest cascade and default nicely